### DISCUSSION TITLE
About the CanIf_Init function

### DISCUSSION THREAD

**Discussion by HMTCT** [autoas/ssas-public#23] [Q&A]
Hello, I wonder if this function hasn't been implemented yet or it just does nothing here.
![image](https://github.com/autoas/ssas-public/assets/89897892/c0993b53-5971-49f5-a0a4-a597647e7cef)

Hope to hear from you soon,

Best regards.

**Comment by parai** [comment]
it's a simple version of CanIf, just empty.
